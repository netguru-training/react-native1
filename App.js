import React from "react";
import {
  StyleSheet,
  Text,
  View,
  ListView,
  TextInput,
  TouchableOpacity,
  AsyncStorage
} from "react-native";
import TaskList from "./src/components/TaskList";
import MainList from "./src/components/MainList";

import MainNavigator from './src/MainNavigator'

export default class App extends React.Component {
  

  render() {
  return (
    <MainNavigator/>

    );
  }
}
