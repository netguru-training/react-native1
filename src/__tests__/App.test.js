import "react-native";
import React from "react";
import App from "../../App";

import { shallow } from "enzyme";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

const wrapper = shallow(<App />);
describe("Rendering", () => {
  test("", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
