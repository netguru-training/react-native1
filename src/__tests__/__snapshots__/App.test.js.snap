// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering  1`] = `<NavigationContainer />`;
